apps/block_scout_web/lib/block_scout_web/templates/smart_contract/_functions.html.eex

```diff
@@ -52,7 +52,11 @@
           end
         %>
         <%= render BlockScoutWeb.SmartContractView, "_pending_contract_write.html" %>
-        <form class="form-inline" data-function-form data-action="<%= if writable?(function), do: :write, else: :read %>" data-type="<%= @contract_type %>" data-url="<%= smart_contract_path(@conn, :show, Address.checksum(@address.hash)) %>" data-contract-address="<%= @address.hash %>" data-contract-abi="<%= @contract_abi %>" data-implementation-abi="<%= @implementation_abi  %>" data-chain-id="<%= Explorer.Chain.Cache.NetVersion.get_version() %>">
+        <%= if @contract_type == "proxy" do %>
```

Comment:
Looks like this conditional logic can be moved to the controller that would then pass the correct address to the template as a parameter.

Reply:
I will move it there